functions/go/gatekeeper/README.md

```diff
@@ -4,29 +4,93 @@
 
 <!--mdtogo:Short-->
 
-Validate the KRM resources using [Gatekeeper] policies.
+The `gatekeeper` function wraps the [`Gatekeeper`] policy enforcement engine to
+evaluate [Gatekeeper] constraints to validate the KRM resources.
 
 <!--mdtogo-->
 
-### FunctionConfig
+It provides a very flexible and powerful way to validate KRM resources against
+custom policies.
+
+You can learn more about how to use the `Gatekeeper` project [here][howto]. The
+concept document about the `OPA Constraint Framework` can be found
+[here][concept].
```

Comment:
An alternate wording for second sentence:

The policies are expressed using `OPA Constraint Framework`, you can read more about it [here](...).